x-pack/plugins/security_solution/common/endpoint/data_generators/fleet_action_generator.ts

```diff
@@ -35,6 +76,45 @@ export class FleetActionGenerator extends BaseDataGenerator {
     );
   }
 
+  /** Generate an OSQuery Action */
+  generateOSQueryAction(overrides: DeepPartial<OSQueryAction> = {}): OSQueryAction {
```

Comment:
This is probably a weird thing to have. I can see using this locally, for dev. But I don't think this is something we should ship inside security_solution.

Reply:
I should remove these methods before this PR is merged as this was added purely for testing manually with dummy data.

Reply:
done c5227c092a3cc4d1fe20ae8d1de26cc0bc0194ff